library/ortools/Deploy.kt

```diff
@@ -0,0 +1,118 @@
+package library.ortools
+
+import org.zeroturnaround.exec.ProcessExecutor
+import org.zeroturnaround.exec.ProcessResult
+import java.math.BigInteger
+import java.nio.charset.StandardCharsets.UTF_8
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.Paths
+import java.security.MessageDigest
+import java.util.*
+
+fun main() {
+    val username = Objects.requireNonNull(
+            System.getenv("DEPLOY_MAVEN_USERNAME"),
+            "username should be passed via \$DEPLOY_MAVEN_USERNAME env variable"
+    )
+    val password = Objects.requireNonNull(
+            System.getenv("DEPLOY_MAVEN_PASSWORD"),
+            "password should be passed via \$DEPLOY_MAVEN_PASSWORD env variable"
+    )
+
+    val repository = "https://repo.grakn.ai/repository/maven"
+    val otGroupId = "com/google/ortools"
+    val otVersion = "8.0.8283"
+
+    /*
+     * Google OT Darwin artifacts
+     */
+    val otDarwin_ArtifcatId = "ortools-darwin"
+    val otDarwin_PomFile = Paths.get("external", "google_ortools_osx", "pom-runtime.xml")
```

Comment:
hang on, i thought there's one JNI wrapper jar/pom/sources that are not platform dependent, and there's the set of ones specifically for darwin/osx... why are both of these listed as `_osx`?

Also, will darwin builds work for any mac? Is it better to keep darwin in the name to be obvious what version its expected to work (and upwards)?